### DISCUSSION TITLE
Get total plots across multiple harvesters

### DISCUSSION THREAD

**Discussion by stavrt** [Chia-Network/chia-blockchain#4535] [General]
I successfully created multiple harvesters and a master full node running. Okay great, but the GUI does not show a total # of plots created. Is there a way to currently do it? If not I guess that's a future feature? I use plotman to create the plots.

**Comment by lvcivs** [comment]
This has been requested repeatedly. It makes a lot of sense to have this feature and I am pretty sure they will eventually get around to adding it, it's just that the devs have more pressing things right now, like getting pool support out.

**Reply by stavrt** [reply]
One workaround I just did was.... to network the drives and share them to the master node. The master node connects to the farmer servers, gets each network drive, maps it and aggregates the plots. Seems to work. Any downside to this?

**Reply by keeperlink** [reply]
The downside is that querying time of plots that passed the filter increases significantly over the network drives and on some occasions you may hit over 5 seconds response warning. Check you logs.

**Reply by stavrt** [reply]
I only have 94 plots right now but here is from my logs:
1 plots were eligible for farming 66f045e206... Found 0 proofs. Time: 1.24070 s. Total 94 plots
1 plots were eligible for farming 66f045e206... Found 0 proofs. Time: 0.75697 s. Total 94 plots
1 plots were eligible for farming 66f045e206... Found 0 proofs. Time: 0.90956 s. Total 94 plots

I looked at my logs and the longest one was 1.9seconds and average is 1s
